IOhandler.cs

```diff
@@ -244,36 +249,17 @@ private void setValueOfMemberField(FieldInfo i_memberField, GarageLogic.Vehicle
             string nameOfMemberField = getNameOfMemberField(i_memberField);
             Type fieldType = i_memberField.GetType();
             string fieldOutName = string.Format("Enter the value for field - {0}:{1}", nameOfMemberField, Environment.NewLine);
-            if (fieldType.IsEnum) // is number
+            if (fieldType.IsEnum) 
             {
-                System.Array enumValues = System.Enum.GetValues(fieldType); //get enum options
+                System.Array enumValues = System.Enum.GetValues(fieldType); 
 
                 m_UI.ShowOptionFromArray(fieldOutName, enumValues);
 
                 int intValue = m_UI.GetIntNumber();
                 PropertyInfo propertyInfo = i_Vehicle.GetType().GetProperty(i_memberField.Name);
                 propertyInfo.SetValue(i_Vehicle, Convert.ChangeType(intValue, i_memberField.FieldType), null);
```

Comment:
Dont remove 
//System.Type enumUnderlyingType = System.Enum.GetUnderlyingType(fieldType);

we might still need it